handler/src/test/java/io/netty/handler/ssl/SslHandlerTest.java

```diff
@@ -560,4 +564,127 @@ public void operationComplete(Future<Channel> future) {
             ReferenceCountUtil.release(sslClientCtx);
         }
     }
+
+    @Test(timeout = 30000)
+    public void testCompositeBufSizeEstimationGuaranteesSynchronousWrite() throws CertificateException, SSLException {
```

Comment:
SslHandler is involved in the estimation process so it has a role to play in this failure.